docs/general/session.md

```diff
@@ -35,4 +35,36 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Events
+
+### Session Start Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.start`
+
+For instrumentation that tracks users behavior during user sessions, a `session.start` event MUST be emitted
+every time a session is created. When a new session is created as a continuation of a prior session,
+the `session.previous_id` SHOULD be included in the event. The values of `session.id` and `session.previous_id`
+MUST be different.
+
+| Body field                                                    | Type   | Description                                          | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) |
+|---------------------------------------------------------------|--------|------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| [`session.id`](/docs/attributes-registry/session.md)          | string | The ID of the new session being started.             | `Required`                                                                                            |
+| [`session.previous_id`](/docs/attributes-registry/session.md) | string | The previous `session.id` for this user, when known. | `Conditionally Required`                                                                              |
+
+### Session End Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.end`
+
+For instrumentation that tracks users behavior during user sessions, a `session.end` event SHOULD be emitted
+every time a session ends. When a session ends and continues as a new session, this event SHOULD also be
+emitted prior to the `session.start` event.
```

Comment:
If a start event arrives before an end event for the previous session, how should the server handle it? Is receiving a start event with a previous session an indication that that previous session has ended?

If the answer is yes, then what I raised in the previous PR would be handled with these two events - end can be used if there is no new session begins, and a start can be used if one does.

If that is the correct interpretation, maybe we can make that explicit in this definition?